internal/pkg/log/api_test.go

```diff
@@ -14,31 +14,29 @@ func TestApiLogger_VerboseFalse(t *testing.T) {
 
 	out := new(bytes.Buffer)
 	stdLogger := stdLog.New(out, "[std-prefix]", 0)
-	logger := NewAPILogger(stdLogger, "[old-prefix] ", false)
+	logger := NewAPILogger(stdLogger, false).AddPrefix("[prefix1]")
```

Comment:
Tu je nejaky existujuci test, ktory som upravil.

Use-case na zmenu prefixu nemame, preto je tam iba metoda `AddPrefix` a to sa aj testuje.